tests/BasicIterator/BasicIterator.sol

```diff
@@ -7,7 +7,7 @@ contract BasicIterator {
     address creator;                  // reserve one "address"-type spot
     uint8[10] integers;               // reserve a chunk of storage for 10 8-bit unsigned integers in an array
 
-    function BasicIterator() 
```

Comment:
dah 😅 @j-f1 as we "translate" `function ContractName` => `constructor`

Reply:
The plugin is doing that right now? I mean, converting `function Contract` to `constructor`? Because that shouldn't be prettier's responsibility IMO.

Reply:
100% agree. Apparently it does that. See if I can fix it.

Reply:
I guess the parser adds the `isConstructor` metadata to functions named as the contract, and the printer function uses that to print `constructor`.